_data/ms10398/ms10398.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Mohit Sharma
+institution: Chitkara University
+profile_pic: ms10398_resized.jpeg
+quote: "Too much data :D, This seems like a joke just like my life"
```

Comment:
Please do not include quotes(")

Reply:
Though github shows it same even if you put single quote double quote or nothing but thanks for the suggestion 😃